hack/release.go

```diff
@@ -0,0 +1,131 @@
+/*
+2023 NVIDIA CORPORATION & AFFILIATES
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
some general comments:

the need to customize a new field would require a change to this file (Release struct). 
this is not ideal.

IMO this should generically support getting a config file with values and apply them on a template

also im not a fan of having multiple sources of a value (file and env variable) is that really needed ?

some alternatives which you can use if you like:
https://jinja.palletsprojects.com/en/3.1.x/templates/
https://github.com/hairyhenderson/gomplate

Reply:
> some general comments:
> 
> the need to customize a new field would require a change to this file (Release struct). this is not ideal.
> 
> IMO this should generically support getting a config file with values and apply them on a template
we've got a very limited list of items to support, so it shouldn't be an issue

> also im not a fan of having multiple sources of a value (file and env variable) is that really needed ? for simplicity there need to be just a single source of truth IMO.
collect everything in file is helpful to quick check what do we have in our dependencies. ENV variables are only needed for quick override of them during builds but we can omit this feature

> some alternatives which you can use if you like: https://jinja.palletsprojects.com/en/3.1.x/templates/ https://github.com/hairyhenderson/gomplate

jinja2 is a really good template engine but I won't to bring python3 and jinga to the project now.  thanks for the reference to `gomplate`, I'll take a look on it